Source/r_main.c

```diff
@@ -601,13 +601,46 @@ void R_SetupFrame (player_t *player)
   validcount++;
 }
 
+//
+// R_ShowStats
+//
+
+int rendered_visplanes, rendered_segs, rendered_vissprites;
+boolean rendering_stats;
+
+static void R_ShowStats(void)
+{
+  static int saved_tick, frame_count;
+  int tick = I_GetTimeMS();
```

Comment:
Hm, another tic counting function. How about we move this to `P_PlayerThink()` just as I am going to do with the IDMYPOS cheat and update once per tic? We could also go the exact same way and update the stats for only 10 seconds.

Reply:
What is `frame_count` for?

Reply:
Aletrnatively, we could count `gametic` or `leveltime` and update after `TICRATE` instead of calling into the low-level milisecond function.

Reply:
In PrBoom+, there was a frame rate counter here. We can restore it and remove another FPS function.

Reply:
But there is nothing wrong with our frame counter, either.